salt/observer/cron.sls

```diff
@@ -2,5 +2,5 @@ daily-metrics-import:
     cron.present:
         - user: {{ pillar.elife.deploy_user.username }}
         - identifier: daily-metrics-import
-        - name: cd /srv/observer/ && ./manage.sh load_from_api --target elife-metrics
+        - name: cd /srv/observer/ && ./daily.sh
```

Comment:
is this already in `master` or does it need to wait?

Reply:
this PR depends on https://github.com/elifesciences/observer/pull/37